pkg/integration/tests/interactive_rebase/squash_fixups_above_first_commit.go

```diff
@@ -0,0 +1,49 @@
+package interactive_rebase
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var SquashFixupsAboveFirstCommit = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Squashes all fixups above the first (initial) commit.",
+	ExtraCmdArgs: "",
+	Skip:         false,
+	SetupConfig:  func(config *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
+		shell.
+			CreateNCommits(2).
+			CreateFileAndAdd("fixup-file", "fixup content")
+	},
+	Run: func(t *TestDriver, keys config.KeybindingConfig) {
+		t.Views().Commits().
+			Focus().
+			Lines(
+				Contains("commit 02"),
+				Contains("commit 01"),
+			).
+			NavigateToListItem(Contains("commit 01")).
+			Press(keys.Commits.CreateFixupCommit).
+			Tap(func() {
+				t.ExpectPopup().Confirmation().
+					Title(Equals("Create fixup commit")).
+					Content(Contains("Are you sure you want to create a fixup! commit for commit")).
+					Confirm()
+			}).
+			NavigateToListItem(Contains("commit 01")).
```

Comment:
It's unfortunate that we have to navigate down again here; it would be nice if the selection was adjusted down by one after creating a fixup commit, so that you stay on the same commit. I guess that's an independent fix though.

Reply:
I agree, moving the cursor down by one would be good.